packages/release/aws-config.conf

```diff
@@ -0,0 +1,3 @@
+[Service]
+# Allow system services load ${HOME}/.aws/config
```

Comment:
This is a bit of a misleading comment:
* it only applies to system services using the AWS SDK for Go
* those services can already set this variable and load the config if they want

I sort of understand the rationale for applying this as a system-wide setting but it might be good to state it here.